runtime/doc/vim9.txt

```diff
@@ -1804,6 +1804,9 @@ Here's an example: >
 	echo s1->add('a')   # output: "['a']"
 	echo s2->add('a')   # E1130: Can not add to null list
 <
+NOTE: the un-initialized var their type or value maybe unusual like other
+coding languages, specially e.g "var a: any" and "var s: string", see `E1134`.
```

Comment:
The example has `E1130` which is the same paragraph that you mention here `E1134`. This seems redundant.

This new `NOTE` doesn't belong right here. There is no `any` in the example.

As mentioned in #13433 typename can give confusing information. There may be a place in help to discuss this, but it is not at this specific location.

Reply:
no, it is just explained or giving a note about why that Err happened.